### ISSUE TITLE
UI Component - Text Combination

### ISSUE THREAD

**Issue by briansztamfater** [status-im/status-react#15413]
Create the quo2 component text combination as per designs:

https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=8737-151875&t=YlBduVIByIv6ifRn-0
Should be implemented according to best practices
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md

Include component spec tests
https://github.com/status-im/status-mobile/blob/develop/doc/component-tests-overview.md

Communicate with design team before starting to make sure there is no unknown requirements

Blur variant is out of scope, only the first 8 variants are to be implemented, there is already some work done on the basic variants, please reuse as much as already existing components as possible.

**Comment by rfahernandez** [comment]
Hi @briansztamfater, I'm getting an error while trying to open the Figma link, could you check is the correct one?

**Comment by siddarthkay** [comment]
Dear @rfahernandez : Thank you for your interest in contributing further, we generally reserve issues with label "bounty" for candidates. Meanwhile feel free to browse issues and pick up any other issue that does not contain this label 🙏🏻

**Comment by rfahernandez** [comment]
Hi @siddarthkay, thanks for the info, will search for other issue without this label then 👋

**Comment by baibhavKumar1** [comment]
@siddarthkay , how to be a candidate?

**Comment by siddarthkay** [comment]
> how to be a candidate

check : https://jobs.status.im/

**Comment by baibhavKumar1** [comment]
Thanks for updating.
